docs/website/docs/plus/features/projects.md

```diff
@@ -0,0 +1,148 @@
+1. a dlt REST API source whose parameters are passed within the manifest
+2. a github source whose source code inside `sources/github.py` is referenced
+  
+```yaml
+sources:
+  pokemon:
+    type: dlt.sources.rest_api.rest_api
+    client:
+      base_url: https://pokeapi.co/api/v2/
+    resource_defaults:
+      endpoint:
+        params:
+          limit: 1000
+    resources:
+      - pokemon
+      - berry
+
+  github:
+    type: dlt_example_project.github.source
+```
+#### Destinations  
+  
+The destinations section defines dlt destinations in a similar way to how you would define them in a pure Python dlt project. As with sources, you can also reference custom implementations of destinations from inside the `destinations/` folder.
+  
+```yaml  
+destinations:
+    duckdb:
+        type: duckdb
+```
+
+#### Pipelines 
+
+Pipelines can be used to load data from sources to destinations. The pipeline defined below loads data from the github source to a dataset named "github_events_dataset" inside the duckdb destination.
+
+```yaml
+  github_pipeline:
+    source: github
+    destination: duckdb
+    dataset_name: github_events_dataset
+```
+You can declare all arguments of `dlt.pipeline` in this section. For a full list of arguments, refer to the [docstrings](https://github.com/dlt-hub/dlt/blob/71b4975c70d1931750b3245e919a520a2400e870/dlt/pipeline/__init__.py#L30).  
+  
+
+#### Datasets
+
+The datasets section defines datasets that live on a destination (defined in the destinations section). Read more about datasets in dlt+ [here](../core-concepts/datasets.md).  
+  
+```yaml
+datasets:
+    github_events_dataset:
+        "on":
+            - duckdb
+```
+
+#### Caches  
+
+In this section you specify the input table(s) that you want to transform, and the output table(s) that you want to write after performing the transformations. The example below loads the table "events" from the destination dataset "github_events_dataset" into a local cache, then transforms it using the transformations inside the `transformations/` folder, and finally writes two tables back into the dataset "github_events_dataset": the original "events" table, and the transformed "events_aggregated" table. Read more about how local cache is used for transformations [here](../core-concepts/datasets.md).
+  
+```yaml
+caches:
+  github_events_cache:
+    inputs:
+      - dataset: github_events_dataset
+        tables:
+          events: events
+    outputs:
+      - dataset: github_events_dataset
+        tables:
+          events: events
+          events_aggregated: events_aggregated
+```
+
+#### Transformations
+
+Here you specify the settings for your transformations. In the code example we define an arrow-based transformation that will operate on the cache "github_events_cache". It will make use of code in the `transformations/` folder. Read more about how transformations are done [here](../core-concepts/cache.md). 
```

Comment:
```suggestion
Here you specify the settings for your transformations. In the code example we define an arrow-based transformation that will operate on the cache "github_events_cache". It will make use of code in the `transformations/` folder. Read more about how transformations are done [here](../features/transformations/index.md). 
```

Reply:
This is still relevant :)